files/zh-cn/learn/javascript/building_blocks/functions/index.md

```diff
@@ -47,8 +49,8 @@ console.log(newString);
 或者当我们操作一个数组的时候：
 
 ```js
-var myArray = ["I", "love", "chocolate", "frogs"];
-var madeAString = myArray.join(" ");
+const myArray = ["I", "love", "chocolate", "frogs"];
+const madeAString = myArray.join(" ");
 console.log(madeAString);
 // join() 函数接受一个数组，将
 // 所有数组元素连接成一个单一的
```

Comment:
Ditto, we usually don't need to take a line break. If necessary, please follow Chinese customs.

Reply:
fixed, thx!